File: VB/dxPaintValue/Form1.vb
Imports System.Windows.Forms

Namespace dxPaintValue
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            Dim c As [MyClass] = New [MyClass]() With {.Color = "Orange"}
            propertyGrid1.SelectedObject = c
            myPropertyGridControl1.SelectedObject = c
        End Sub
    End Class
End Namespace

File: CS/dxPaintValue/Form1.cs
using System.Windows.Forms;

namespace dxPaintValue {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            MyClass c = new MyClass() { Color = "Orange" };
            propertyGrid1.SelectedObject = c;
            myPropertyGridControl1.SelectedObject = c;
        }
    }
}
